bin/setup

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env ruby
 require "fileutils"
 
-# path to your application root.
 APP_ROOT = File.expand_path("..", __dir__)
+APP_NAME = "sulbib"
```

Comment:
sulbib?

Reply:
yeah, that is the historic name of the application, as shown in `config/application.rb` ... this was auto added by the script, presumably from that module name